examples/threadsafe/tests/bindings/test_threadsafe.kts

```diff
@@ -42,7 +42,7 @@ fun countWhileBusy(busyWait: () -> Int, incrementIfBusy: () -> Int): Int {
 // long enough for us to incrementIfBusy.
 // If the Rust compiler gets too smart, and optimizes the busy wait loop away,
```

Comment:
seems extremely unlikely the compiler will optimize the sleeping version away? ISTM the main risk here now is the speed of the hardware?